_posts/2022-02-18-release-notes.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: post
+date: February 18, 2022
```

Comment:
I think you'll need to change this to the current date if you want it to display in the Federalist preview. Posts with future dates are displayed.

Reply:
well that explains that. :) thx

Reply:
Or you can review the preview tomorrow